h/static/styles/top-bar.scss

```diff
@@ -31,10 +31,15 @@
   padding-right: $h-padding;
 
   height: 100%;
+
+  .signin-control{
```

Comment:
We try to avoid nested selectors, so this declaration should go at the top-level of the file if possible.